CMakeLists.txt

```diff
@@ -1,4 +1,10 @@
 cmake_minimum_required(VERSION 3.0)
+
```

Comment:
Please, refer in the commit message to the function as `dlopen()` (note the parentheses).